src/panels/FlightControlPanel/FlightControlPanel.tsx

```diff
@@ -0,0 +1,141 @@
+import { useEffect } from 'react';
+import { Chip, Container, Group, List, Slider, Switch, Text, Title } from '@mantine/core';
+
+import { useGetBoolPropertyValue, useOpenSpaceApi } from '@/api/hooks';
+import { Tooltip } from '@/components/Tooltip/Tooltip';
+import { setSettings } from '@/redux/flightcontroller/flightControllerSlice';
+import { useAppDispatch, useAppSelector } from '@/redux/hooks';
+import {
+  subscribeToProperty,
+  unsubscribeToProperty
+} from '@/redux/propertytree/properties/propertiesMiddleware';
+import { RollFrictionKey, RotationalFrictionKey, ZoomFrictionKey } from '@/util/keys';
+
+export function FlightControlPanel() {
+  const luaApi = useOpenSpaceApi();
+  const dispatch = useAppDispatch();
+
+  const isControllerEnabled = useAppSelector(
+    (state) => state.flightController.settings.isEnabled
+  );
+  const mouseScaleFactor = useAppSelector(
+    (state) => state.flightController.settings.inputScaleFactor
+  );
+
+  const rotationFriction = useGetBoolPropertyValue(RotationalFrictionKey) ?? false;
+  const zoomFriction = useGetBoolPropertyValue(ZoomFrictionKey) ?? false;
+  const rollFriction = useGetBoolPropertyValue(RollFrictionKey) ?? false;
+
+  useEffect(() => {
+    function subscribeTo(uri: string) {
+      dispatch(subscribeToProperty({ uri }));
+    }
+    function unsubscribeTo(uri: string) {
+      dispatch(unsubscribeToProperty({ uri }));
+    }
+
+    subscribeTo(RotationalFrictionKey);
+    subscribeTo(ZoomFrictionKey);
+    subscribeTo(RollFrictionKey);
+
+    return () => {
+      unsubscribeTo(RotationalFrictionKey);
+      unsubscribeTo(ZoomFrictionKey);
+      unsubscribeTo(RollFrictionKey);
+      dispatch(setSettings({ isEnabled: false }));
+    };
+  }, [dispatch]);
+
+  function toggleEnable() {
+    dispatch(setSettings({ isEnabled: !isControllerEnabled }));
+  }
+
+  function toggleRotation() {
+    luaApi?.setPropertyValueSingle(RotationalFrictionKey, !rotationFriction);
+  }
+
+  function toggleZoom() {
+    luaApi?.setPropertyValueSingle(ZoomFrictionKey, !zoomFriction);
+  }
+
+  function toggleRoll() {
+    luaApi?.setPropertyValueSingle(RollFrictionKey, !rollFriction);
+  }
+
+  const tooltipContent = (
+    <Container>
+      <Text>Interact with the area to control the camera.</Text>
```

Comment:
With the change from the area in the panel to the entire available rendering, this comment is no longer clear and needs updating.  Maybe: 

```suggestion
      <Text>Interact with the OpenSpace rendering to control the camera.</Text>
```
? Or change the description entirely to make it more clear?

Reply:
Added a visible border to the interaction, text reads "Interact with the highlighted area to ..."